src/main/java/seedu/duke/Duke.java

```diff
@@ -29,21 +33,51 @@ public static void main(String[] args) {
         /**
          * Calorie List and List
          */
+        /*
         // Example code to use calorie list.
         ActivityMap calList = new ActivityMap();
 
-        LocalDateTime adatetime = LocalDateTime.of(2015,
-                Month.JULY, 29, 19, 30, 40);
+        LocalDateTime adatetime = LocalDateTime.of(2015, Month.JULY, 29, 19, 30, 40);
 
         calList.addActivity(adatetime, 10);
         calList.addActivity(adatetime, 11);
         calList.addActivity(adatetime, 12);
 
         //System.out.println(calList.toString(aDateTime));
         //System.out.println("Size of activity list: " + calList.getSizeOfActivityList(aDateTime));
+        */
+
         /**
          * Add exercise/food with their respective calories
          */
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+        int numOfActivities = INITIALISE;
+        try {
+            if (userInput.startsWith("add f/")) {
+                int calorieIndex = userInput.indexOf("c/");
+                int calories = Integer.parseInt(userInput.substring(calorieIndex + 3));
+                userInput = userInput.substring(6, calorieIndex - 1);
+                Activity item = new Food(userInput, calories);
+                activities.add(item);
+                System.out.println("\t" + activities.get(numOfActivities++).toString());
+            } else if (userInput.startsWith("add e/")) {
+                int calorieIndex = userInput.indexOf("c/");
+                int calories = Integer.parseInt(userInput.substring(calorieIndex + 3));
+                userInput = userInput.substring(6, calorieIndex - 1);
+                Activity item = new Exercise(userInput, calories);
+                activities.add(item);
+                System.out.println("\t" + activities.get(numOfActivities++).toString());
+            }
+        } catch (StringIndexOutOfBoundsException e) {
+            System.out.println("Something went wrong!! I do not understand what you mean.\n"
+                    + "There could be an error in the way of input.\n"
+                    + "Please do input 'help' for the commands and their respective input format.");
+        } catch (Exception e) {
+            System.out.println("Something went wrong!! I do not understand what you mean.\n"
+                    + "There could be an error in the way of input.\n"
+                    + "Please do input 'help' for the commands and their respective input format.");
+        }
```

Comment:
Might have to SLAP this later on, once the base parser class Is up

Reply:
noted